src/game.py

```
@@ -0,0 +1,31 @@
+from player import Player
+from airport import Airport
```

Ruff (0.8.2)

`airport.Airport` imported but unused

Remove unused import: `airport.Airport`

(F401)